src/core/server/config/schema/types/__tests__/any_type.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { schema } from '../..';
+
+test('works for any value', () => {
+  expect(schema.any().validate(true)).toBe(true);
+  expect(schema.any().validate(100)).toBe(100);
+  expect(schema.any().validate('foo')).toBe('foo');
+  expect(schema.any().validate({ foo: 'bar', baz: 2 })).toEqual({ foo: 'bar', baz: 2 });
```

Comment:
This tests boolean, number, string, object. what about null and undefined?

Reply:
Yep, added a couple of `expect`'s for `null`, thanks!